addons/crm/views/res_config_settings_views.xml

```diff
@@ -6,6 +6,7 @@
         <field name="model">res.config.settings</field>
         <field name="priority" eval="5"/>
         <field name="inherit_id" ref="base.res_config_settings_view_form"/>
+        <field name="groups_id" eval="[(4, ref('sales_team.group_sale_salesman'), 0)]"/>
```

Comment:
The whole div ``app_settings_block`` has sales_team.group_sale_manager group. I don't think it is necessary to add it in view, especially this may break inheriting views.

Reply:
It's not enough, because the div group only affect visibility, but not the data that is loaded from backend. Thus, the permissions error would still happen. Otherwise, this bug wouldn't exist.

That's why the permission must be in the view itself.

Reply:
And adding it on the crashing fields ?

Reply:
It has the same effect. Once the view is loaded, all defined fields are loaded. I think it's done to cover the case when `invisible` is dynamic.

In any case, if people aren't getting this error, is because they have the permissions, so they shouldn't see any behavioural change.